test/extended/oauth/http1.go

```diff
@@ -17,6 +17,15 @@ var _ = g.Describe("[sig-auth][Feature:OAuthServer] OAuth server", func() {
 
 	oc := exutil.NewCLI("oauth")
 
+	g.BeforeEach(func() {
+		microShift, err := exutil.IsMicroShiftDeployment(oc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if microShift {
+			// MicroShift does not have the oauth well known data path in the API Server
+			g.Skip("Skipped in MicroShift")
```

Comment:
Would it make more sense to extend `g.Describe` with `[apigroup:oauth.openshift.io]` instead?

Reply:
Done